DigitalLearningSolutions.Web.AutomatedUiTests/SeleniumServerFactory.cs

```diff
@@ -69,7 +69,8 @@ private static IConfigurationRoot GetConfigForUiTests()
             return new ConfigurationBuilder()
                 .SetBasePath(Directory.GetCurrentDirectory())
                 .AddJsonFile("appsettings.SIT.json")
-                .AddEnvironmentVariables("DlsRefactor_")
+                .AddEnvironmentVariables("DlsRefactor_")
+                .AddUserSecrets(typeof(Startup).Assembly)
```

Comment:
This gives the UI tests access to the Secrets from the Web project so that the UI tests can pick up the API key locally.